tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -488,6 +488,31 @@ file_cmp (const gchar * f1, const gchar * f2)
   return cmp;
 }
 
+/**
+ * @brief Wait until the change in pipeline status is done
+ * @return ML_ERROR_NONE success, ML_ERROR_UNKNOWN if failed, ML_ERROR_TIMED_OUT if timeout happens.
+ */
+int
+waitPipelineStateChange (ml_pipeline_h handle, ml_pipeline_state_e state,
+    guint timeout_ms)
+{
+  int status = ML_ERROR_UNKNOWN;
+  guint counter = 0;
+  ml_pipeline_state_e cur_state = ML_PIPELINE_STATE_NULL;
+
+  do {
+    status = ml_pipeline_get_state (handle, &cur_state);
+    EXPECT_EQ (status, ML_ERROR_NONE);
+    if (cur_state == ML_PIPELINE_STATE_UNKNOWN)
```

Comment:
status != ML_ERROR_NONE || cur_state == ML_PIPELINE_STATE_UNKNOWN

Reply:
Thank you for your comment. 
There seems to be no problem with the proposed revision.
However, if 'ml_pipeline_get_state'  fails, the actual return values vary from ML_ERROR_INVALID_PARAMETER, ML_ERROR_STREAMS_PIPE and etc. 
So, ML_ERROR_UNKNOWN is not an exact return value.